### PR TITLE
Minor README updates

### PR DESCRIPTION
**Pull request by WinFan3672**
I fixed some typos.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,4 +16,4 @@ Feel free to submit pull requests to the game. If merged, they will be shipped i
 - Scenes are in `/Assets/Scenes`. The game's entry-point is `/Assets/Scenes/Bootstrap`
 
 ## SDSH scripts
-Many parts of the game use `sdsh` (Socially Distant SHell) scripts. These are `.sh` files in the bundled assets directory. They use a POSIX-like shell scripting language that the game interprets. When scripting gameplay encounters like missions and NPC interactions, always prefer `sdsh` over C#.
\ No newline at end of file
+Many parts of the game use `sdsh` (Socially Distant Shell) scripts. These are `.sh` files in the bundled assets directory. They use a POSIX-like shell scripting language that the game interprets. When scripting gameplay encounters like missions and NPC interactions, always prefer `sdsh` over C#.
```
Comment (TheCourierNV) [inline README.md]: I believe that one was intended

Comment (WinFan3672) [reply README.md]: If so, wouldn't **bold** be more appropriate?

Comment (TheCourierNV) [reply README.md]: Bolding it is probably fine

Comment (WinFan3672) [reply README.md]: Fixed: bde86cac2c2744bb360fe91b32645ea24c23adb0

File: README.md [hunk 2]
```diff
@@ -16,4 +16,4 @@ Feel free to submit pull requests to the game. If merged, they will be shipped i
 - Scenes are in `/Assets/Scenes`. The game's entry-point is `/Assets/Scenes/Bootstrap`
 
 ## SDSH scripts
-Many parts of the game use `sdsh` (Socially Distant SHell) scripts. These are `.sh` files in the bundled assets directory. They use a POSIX-like shell scripting language that the game interprets. When scripting gameplay encounters like missions and NPC interactions, always prefer `sdsh` over C#.
\ No newline at end of file
+Many parts of the game use `sdsh` (**S**ocially **D**istant **Sh**ell) scripts. These are `.sh` files in the bundled assets directory. They use a POSIX-like shell scripting language that the game interprets. When scripting gameplay encounters like missions and NPC interactions, always prefer `sdsh` over C#.
```
Comment (TheCourierNV) [inline README.md:19]: Also, at this point it would probably be best to have both bolded and uppercase

**Comment by acidiclight** [comment]
Closing as this PR was made before the DCO enforcement was set up.

**Pull request closed without merging**